src/main/java/com/verisign/storm/metrics/graphite/GraphiteConnectionFailureException.java

```diff
@@ -16,9 +16,9 @@
 
 import java.io.IOException;
 
-public class GraphiteConnectionAttemptFailure extends IOException {
+public class GraphiteConnectionFailureException extends IOException {
```

Comment:
Should we rename `GraphiteConnectionFailureException` to `BackendConnectionFailureException` or simply `ConnectionFailureException`?